ai_services/anomaly_detection/data_preprocessing_examples/oci_data_flow_based_examples/setup-archive.sh

```diff
@@ -1,9 +1,13 @@
 #!/bin/bash
 pip install wheel setuptools
 python setup.py bdist_wheel
-if [ ! -d "dtst" ]
+if [ -d "dist" ]
 then
+    cp requirements.txt dist
     cd dist
+    if [ -f "archive.zip" ]; then
+        rm archive.zip
+    fi
     docker pull phx.ocir.io/oracle/dataflow/dependency-packager:latest
     docker run --rm -v $(pwd):/opt/dataflow --pull always -it phx.ocir.io/oracle/dataflow/dependency-packager:latest -p 3.6
```

Comment:
[Minor] Please add pip.conf too and update the command because the distribution might end up generating *archive.zip* with non-public packages too. Also, [line 11](https://github.com/bug-catcher/oci-data-science-ai-samples/pull/8/files#diff-ba11fb1eb11ea547c0f15e52419770f79c7a8aad73f2b0142ece2667c41dbd83R11) can be skipped since the docker run commands includes it.
```suggestion
    docker run --rm -v $(pwd)/pip.conf:/etc/pip.conf  -v $(pwd)/:/opt/dataflow --pull always -it phx.ocir.io/oracle/dataflow/dependency-packager:latest -p 3.6
```